corehq/apps/reports/standard/maps.py

```diff
@@ -251,11 +252,17 @@ def show_in_navigation(cls, domain=None, project=None, user=None):
 class ElasticSearchMapReport(GetParamsMixin, GenericTabularReport, GenericMapReport):
 
     report_template_path = "reports/async/maps.html"
-    report_partial_path = "reports/async/partials/maps.html"
+    report_partial_path = "reports/bootstrap3/partials/base_maps.html"
```

Comment:
once I moved the js files to base rather than the async template, there was no difference between `async/partials/maps.html` and `maps_base.html`